### PR TITLE
source-quora-ads contribution from bala-ceg

### PR DESCRIPTION
**Pull request by bala-ceg**
## What

This PR adds a new connector for source Quora Ads (source-quora-ads).

Contributor-provided connector description:

Quora Ads API Connector allows you to sync advertising data between Quora Ads and external platforms, such as analytics tools, CRMs, or marketing dashboards. Quora Ads is a platform that enables businesses to reach relevant audiences through targeted advertising on Quora, helping drive brand awareness, engagement, and conversions. This connector streamlines the process of retrieving key ad performance data, making it easier to monitor campaigns and optimize marketing strategies.
## Details

<details><summary>Expand for list of spec properties and streams</summary>

### Configuration

| Input | Type | Description | Default Value |
|-------|------|-------------|---------------|
| `campaign_id` | `string` | Campaign Id.  |  |
| `ad_sets` | `string` | adsets.  |  |
| `client_id` | `string` | Client ID.  |  |
| `client_secret` | `string` | Client secret.  |  |
| `client_refresh_token` | `string` | Refresh token.  |  |

### Streams
| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
|-------------|-------------|------------|---------------------|----------------------|
| current_user |  | DefaultPaginator | ✅ |  ❌  |
| accounts | accountId | DefaultPaginator | ✅ |  ❌  |
| leadgenforms |  | DefaultPaginator | ✅ |  ❌  |
| campagins |  | DefaultPaginator | ✅ |  ❌  |
| adsets |  | DefaultPaginator | ✅ |  ❌  |

</details>

## Reviewer checklist
- [ ] Run `/format-fix` to resolve any formatting errors
- [ ] Help the contributor upload an icon that follows our icon design guidelines
- [ ] Click into the CI workflows that wait for a maintainer to run them, which should trigger CI runs

### CODE REVIEW

File: docs/integrations/sources/quora-ads.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Quora Ads
+Quora Ads API Connector allows you to sync advertising data between Quora Ads and external platforms, such as analytics tools, CRMs, or marketing dashboards. Quora Ads is a platform that enables businesses to reach relevant audiences through targeted advertising on Quora, helping drive brand awareness, engagement, and conversions. This connector streamlines the process of retrieving key ad performance data, making it easier to monitor campaigns and optimize marketing strategies.
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `api_key` | `string` | API Key.  |  |
```
Comment (natikgadzhi) [inline docs/integrations/sources/quora-ads.md:8]: Link to how to obtain the api key pretty pelase.

**Review by natikgadzhi — CHANGES_REQUESTED** [review]
@marcosmarxm any ad platform connectors without actual converstion data and spend data are not really useful. With the high volume of data, those are usually especially sensitive to incremental reads.

Very often, performance reporting data would be in, well, a report.

**Comment by marcosmarxm** [comment]
Hello 👋 Due to the numerous contributions happening, I decided to create this [view](https://github.com/orgs/airbytehq/projects/104/views/3) to help you know when I plan to review your contribution. If your connector has some limited-time credential feel free to reach me out.

**Comment by marcosmarxm** [comment]
All contributions must be merged by Nov-14. I'm going to work to provide a first review by Friday to give users the weekend to fix any comment. Please be aware of this.

Make sure you have pagination properly working, the API doesn't have pagination please left a comment making my review easier.
Make sure have a square svg icon
Make sure credentials are valid.

**Comment by marcosmarxm** [comment]
Not possible to test the connector yet, moving next review to Monday Nov 11. If that time I can't run proper tests I'm going to decline the contribution.

**Comment by bala-ceg** [comment]
Hi @marcosmarxm , it is in my anonther airbyte account with user email as balaji281295@gmail.com

**Comment by marcosmarxm** [comment]
The connector in your workspace isn't working. Please fix that by Monday.

**Comment by bala-ceg** [comment]
@marcosmarxm this connector is ready for review (updated the icon and make sure all streams are working) and it is my in airbyte connector of email address - balaji281295@gmail.com

**Comment by marcosmarxm** [comment]
Thanks @bala-ceg but Quora Ads doens't look a good candidate to build using UI Builder. It is very similar to Google Ads, Bing Ads, Facebook Ads. Which demand a lot of complex API calls / parameters and transformation to the data to make sense to end user. Also the account used doesn't much data which can't provide much to use analyze. Thanks for the contribution but for that reason I'm declining it.

**Pull request closed without merging**